com.editor.gamma.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id": "com.editor.gamma",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "/app/bin/gamma",
+    "finish-args": [
+    	"--share=network",
+    	"--socket=x11",
+		"--share=ipc",
+		"--socket=session-bus",
```

Comment:
I'm pretty sure you can restrict this a bit more, please run the app with --log-session-bus to inspect which dbus holes you want to open.

Reply:
when removing `"--socket=session-bus",`:  

I got this warning
```
dbind-WARNING **: 10:20:36.749: AT-SPI: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

Reply:
When removing `"--share=ipc",`, no warnings nor errors but I added it because it is says 

> --share=ipc - share IPC namespace with the host (necessary for X11)

in the docs https://docs.flatpak.org/en/latest/sandbox-permissions.html